runtime/compiler/env/j9method.cpp

```diff
@@ -792,6 +792,15 @@ TR_J9MethodBase::osrFrameSize(J9Method* j9Method)
    {
    if (auto stream = TR::compInfoPT->getStream())
       {
+         {
```

Comment:
I don't see this method being used anywhere, so we should delete it from the master branch as well.